.travis.yml

```diff
@@ -18,4 +18,8 @@ script:
     --exclude ../kubespray/
     --exclude idr-kubernetes-apply.yml
     --exclude idr-reset-users-groups.yml
```

Comment:
Obviously it would make sense to move the above common tasks out of the matrix build instead of running them twice, but it's not clear how from reading the Travis docs, and given the time taken by each re-run I'm not going to attempt it unless someone has a known working example.

Reply:
I do not have any production example either in any of our repos. That being said, I am very interested in converging towards that as we will see the need in other repos. My initial idea would be to try something along the lines of https://docs.travis-ci.com/user/build-stages#build-stages-and-build-matrix-expansion maybe using `molecule syntax` first to prototype and then switch to `molecule test` when the config is working?

Reply:
Thinking about it it might be possible to have a `syntax-only` molecule scenario in which case this the setup here works nicely. Mind if that's in later PR?

Reply:
Looking at the details of the [Travis completion time](https://travis-ci.org/IDR/deployment/builds/463310820)

|  Scenario |  Total time |  Molecule test time |
|-----------|-------------|---------------------|
| publicidr | 24min 46s   | 23min 12s           |
| ftp       |  6min 3s    | 4min36s             |

Previous executions were ~24-25min so I do not think this gets us to a much worse situation since you are effectively testing a completely separate deployment path.